pkg/compose/generate.go

```diff
@@ -222,6 +224,7 @@ func (s *composeService) toComposeNetwork(networks map[string]*network.EndpointS
 	for name, net := range networks {
 		inspect, err := s.apiClient().NetworkInspect(context.Background(), name, network.InspectOptions{})
 		if err != nil {
+			logrus.Warnf("Failed to inspect network %s: %v", name, err)
```

Comment:
@ndeloof 

As described, the containers orchestrated by `docker compose up` in my production environment have experienced network loss [issues](https://github.com/docker/compose/issues/11601) more than once. This problem persists even in the latest version and has troubled me for a long time. I have not been able to find any relevant error logs anywhere to pinpoint the issue. Even in the `cli`, no anomalies are recorded, which is very frustrating for me.

![image](https://github.com/user-attachments/assets/794554df-111c-4c68-a5dc-51adf5a3942f)

Reply:
`generate` command is not designed to be used as a diagnostic tool! You can wrap engine error to give more context, but I don't expect this will give you more than "not found":
```go
  return fmt.Errorf("failed to inspect network %s: %w", networkID, err)
``